### PR TITLE
First pass at a working connection

### PR DESCRIPTION
**Pull request by IngCr3at1on**
DO NOT MERGE

This is most certainly not finished code, figured now would be a good time to discuss the structure a bit. Please comment the shit out of this PR ;)

### CODE REVIEW

**Comment by GrimTheReaper** [comment]
I nearly merged this XD

**Comment by IngCr3at1on** [comment]
LMAO, even with the "DO NOT MERGE" comment? :P

You're supposed to comment on it

File: handlers/clienthandler.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"tiberious/types"
+
+	"github.com/gorilla/websocket"
+	"gopkg.in/redis.v3"
```
Comment (IngCr3at1on) [inline handlers/clienthandler.go:10]: I forgot that I included redis in this PR, I was just sort of playing around with it. We should use some form of database to store basic information that way multiple servers can be configured for the same domain/network.

**Comment by GrimTheReaper** [comment]
Merging to continue progress of the project.

**Pull request merged**